_docs/services/external-domain-service.md

```diff
@@ -202,8 +211,41 @@ cf create-service external-domain domain-with-cdn -c '{"path": "/some/path"}'
 
 ## Update an instance
 
-Updating the plan for an instance is not currently supported by the broker. To migrate from a `domain` plan to a `domain-with-cdn` plan, you must delete the current instance, then create a new one with the new plan.
-
+Not all plans can be updated - see the matrix below for options.
+If you'd like to make an update not supported by the broker, you must delete and re-create your service
+
+<table style="text-align:center;">
+        <tr>
+            <td colspan=2 rowspan=2></td>
+            <th style="text-align:center;" colspan=3 scope="colgroup">New Plan</th>
+        </tr>
+        <tr>
+            <th scope="col">domain</th>
+            <th scope="col">domain-with-cdn</th>
+            <th scope="col">domain-with-org-lb</th>
+        </tr>
+        <tr>
+            <th style="vertical-align:center" rowspan=5 scope="rowgroup">Existing plan</th>
+        </tr>
+        <tr>
+            <th scope="row">domain</th>
+            <td>✅</td>
+            <td>🚫</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <th scope="row">domain-with-cdn</th>
+            <td>🚫</td>
+            <td>✅</td>
+            <td>🚫</td>
+        </tr>
+        <tr>
+            <th scope="row">domain-with-org-lb</th>
+            <td>🚫</td>
+            <td>🚫</td>
+            <td>🚫</td>
+        </tr>
+</table>
```

Comment:
Can we format this using Markdown table syntax for easier maintenance?

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

Reply:
I used an HTML table because Markdown doesn't support merged cells (afaict)

Reply:
Table looks really nice in the Pages preview build